### PR TITLE
Update configuration.md (Add spaces for readability)

### PR DESCRIPTION
**Pull request by DerTimonius**
## Description

This will add just two spaces in the `configuration.md` file to improve formatting a bit, see [this PR comment](https://github.com/prettier/prettier/pull/14063#issuecomment-1525046241)
e.g. `JS(ESM) -> JS (ESM)`

## Checklist

- [ ] I’ve added tests to confirm my change works.
- [ ] (If changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory).
- [ ] (If the change is user-facing) I’ve added my changes to `changelog_unreleased/*/XXXX.md` file following `changelog_unreleased/TEMPLATE.md`.
- [X] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/main/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

File: docs/configuration.md
Comment (fisker) [inline docs/configuration.md:35]:
```suggestion
JS (ES Module)
```

**Review by fisker — APPROVED** [review]

File: docs/configuration.md
Comment (DerTimonius) [reply docs/configuration.md]: Good idea, added this real quick!

Comment (karlhorky) [reply docs/configuration.md]: @fisker The long form of `ESM` is more commonly `ES Modules` (with `s`) - what do you think about that instead?

Comment (fisker) [reply docs/configuration.md]: I don't think so, the config is a single module, why `s`? I'm a native English speaker, I'll let other maintainer decide.

Comment (karlhorky) [reply docs/configuration.md]: This is what I was thinking was your reasoning for it, and technically,  it kind of makes sense

However, in the context of JS, I would argue it's more common for people to look for the project module format (not a description of a single file) - JS module formats are often described as "CommonJS" vs "ES Modules"

Also, people may search for the more common search term "ES Modules" and not be able to find this part of the page

**Comment by DerTimonius** [comment]
@fisker Is there anything left I can do to get this merged?

**Pull request merged**
